packages/notification-service/src/blockscout/transfers.ts

```diff
@@ -117,6 +117,6 @@ export async function handleTransferNotifications() {
 
   const allTransfers = filterAndJoinTransfers(goldTransfers, stableTransfers)
 
-  notifyForNewTransfers(allTransfers, lastBlockNotified)
-  setLastBlockNotified(Math.max(goldTransfersLatestBlock, stableTransfersLatestBlock))
+  await notifyForNewTransfers(allTransfers, lastBlockNotified)
+  await setLastBlockNotified(Math.max(goldTransfersLatestBlock, stableTransfersLatestBlock))
```

Comment:
the last line of a function should never be await, or you're better off just returning the promise object itself.